src/__tests__/starWarsSchema.js

```diff
@@ -105,25 +110,31 @@ const episodeEnum = new GraphQLEnumType({
  *     secretBackstory: String
  *   }
  */
+type CharacterJS = {
```

Comment:
this was already defined in the previous file - import it here rather than duplicating